### PR TITLE
fix: post frontmatter

### PR DESCRIPTION
**Pull request by johnytiago**
## Description - [Trello ticket](https://trello.com/c/AHPXgaX9/759-blog-post-publish-failure)

Yaml stringify frontmatter metadata to prevent it from breaking the build

## Checklist

- [ ] Updating or creating a new page? Consider any SEO requirements such as:
  - [ ] Does the page have an `h1` tag?
  - [ ] Does the page have a correctly formed meta title?
  - [ ] Does the page have a correctly formed meta description?
  - [ ] Does the page have a unique `og:image` tag (if required)?
- [x] checked that this PR resolves the spec provided from trello / this description

If you're unsure how to check these SEO requirements please reach out to your colleagues for help!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -103,7 +103,8 @@
     "turndown": "^5.0.3",
     "tween": "^0.9.0",
     "url-parse": "^1.4.7",
-    "xml2json": "^0.11.2"
```
Comment (jeysal) [inline package.json:106]: We use `yarn`, can you remove the npm `package-lock` file and run `yarn` and commit the `yarn.lock` changes please? :)

File: src/functions/blogMethods/custom-mdx/add-frontmatter.js [hunk 2]
```diff
@@ -6,13 +8,14 @@ const addFrontmatter = ({
   firstPublishedAt,
   authorName,
 }) => `---
-title: ${title}
```
Comment (jeysal) [inline src/functions/blogMethods/custom-mdx/add-frontmatter.js:9]: Can you add a test to `add-frontmatter.test.js` where the title contains a colon please

**Comment by jeysal** [comment]
I will also delete the existing manually edited blog post on YLD.io and trigger a resync when merged to check if it'll work for future posts

**Comment by johnytiago** [comment]
Sounds good! :+1: Thanks

**Pull request merged**